build.ps1

```diff
@@ -119,6 +119,20 @@ Invoke-BuildStep 'Building NuGet.Clients projects - VS15 dependencies' {
 ## Building the VS14 Tooling solution
 Invoke-BuildStep 'Building NuGet.Clients projects - VS14 dependencies' {
         param($Configuration, $ReleaseLabel, $BuildNumber, $SkipRestore, $Fast)
+
+        if (-Not $SkipVS15)
```

Comment:
Let's put it in a separate build step with `skip` value of `(-not $SkipVS14) -and (-not $SkipVS15)`